core/bind/core_bind.cpp

```diff
@@ -646,11 +646,6 @@ uint64_t _OS::get_static_memory_peak_usage() const {
 	return OS::get_singleton()->get_static_memory_peak_usage();
 }
 
-uint64_t _OS::get_dynamic_memory_usage() const {
```

Comment:
was this removed by mistake?

Reply:
I assume yes, as the `MemoryPool` this referred to was also removed: https://github.com/godotengine/godot/pull/36311/files/3205a92ad872f918c8322cdcd1434c231a1fd251#diff-ab1978b28d0ec282d1df67c7903b6191L419-L422